src/components/PickerDay.vue

```diff
@@ -301,6 +303,15 @@ export default {
         this.selectedDate && this.utils.compareDates(this.selectedDate, dObj)
       )
     },
+    localDateString(day) {
+      const options = {
+        weekday: 'long',
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+      }
+      return new Date(day.timestamp).toLocaleDateString(undefined, options)
```

Comment:
Shouldn't the represented label match what will be inserted into the input field?
Or is this common? 

Seems kinda weird to me if some selects something through a screenreader which says `Wednesday, 8. May 2024` but at the end it will tell him that the value of the field is `2024-05-08`.
I guess the build in formatter should be used like in Dateinput.vue

Reply:
That's a good point about the value of `aria-label` not matching what would be inserted into the input field. It would be ideal if their values were the same. 

> I guess the build in formatter should be used like in Dateinput.vue

Are you referring to the `format` property? If so, that makes complete sense.

Reply:
Yes the format property